Sources/Rendering/WebGPU/CellArrayMapper/index.js

```diff
@@ -699,7 +703,27 @@ function vtkWebGPUCellArrayMapper(publicAPI, model) {
         '  var PBR: vec3<f32> = mapperUBO.DiffuseIntensity*kD*diffuse + kS*specular;',
         '  PBR += emission;',
         '  computedColor = vec4<f32>(PBR, mapperUBO.Opacity);',
+        '  //VTK::Light::Impl', // TODO: Find a better way of doing this, because it's ugly
```

Comment:
You can consider conditionally including (or appending) the below shader code here, if you don't want to use this temporary substitution marker.